jni/Android.mk

```diff
@@ -851,6 +851,19 @@ LOCAL_SRC_FILES     := \
 
 include $(BUILD_STATIC_LIBRARY)
 
+
+################################################################################
+# rpgp
+################################################################################
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE    := librpgp
+LOCAL_SRC_FILES := /home/bpetersen/projects/rpgp/target/armv7-linux-androideabi/release/libpgp_ffi.a
```

Comment:
possible to specify this less bjoern-dependent?

Reply:
:)

Reply:
just create a user account with the name `bpetersen` ;)

however, there is much more missing, the whole call of rustc crosscompiling and a bunch of other things ...